From
<https://anonscm.debian.org/cgit/pkg-perl/packages/libgtk2-unique-perl.git/plain/debian/patches/fix_segfault_2nd_instance>
by Simon Kainz <Simon Kainz skainz@debian.org>.

CPAN RT#120115

Index: libgtk2-unique-perl/xs/UniqueApp.xs
===================================================================
--- libgtk2-unique-perl.orig/xs/UniqueApp.xs
+++ libgtk2-unique-perl/xs/UniqueApp.xs
@@ -1,6 +1,8 @@
 #include "unique-perl.h"
 #include <gperl_marshal.h>
 
+G_CONST_RETURN gchar * unique_command_to_string (UniqueApp *app, gint command);
+
 static void
 perl_unique_app_marshall_message_received (
 	GClosure *closure,
